microsoft-edge/devtools-guide-chromium/performance/overview.md

```diff
@@ -0,0 +1,371 @@
+## Overview
+<!-- https://developer.chrome.com/docs/devtools/performance/overview#overview -->
+<!-- https://microsoftedge.github.io/Demos/exploring-the-universe/ -->
+<!-- covers basic nav between the two states, and what to use them for, not how to use either view -->
+
+The **Performance** tool displays local metrics for page interactions, and lets you record CPU performance profiles of your web applications.  Analyze profiles to find potential performance bottlenecks and ways that you can optimize resource use.
+
+
+<!-- ------------------------------ -->
+#### Local metrics for page interactions
+
+Use the **Performance** tool to view Core Web Vitals metrics in the initial, **Local metrics** view.  The home page of the **Performance** tool shows **Local metrics**, which is performance metrics about the rendered webpage:
+
+* Largest Contentful Paint (LCP) - How quickly the main content of the page loaded.
+* Cumulative Layout Shift (CLS) - A measure of the most recent unexpected page layout shift.
+* Interaction to Next Paint (INP) - The responsiveness of the most recent user interaction on the page.
+
+When you initially open the **Performance** tool, Core Web Vitals metrics are displayed, in three cards in a **Local metrics** section:
+
+![The Performance tool home page showing Local metrics](./overview-images/performance-tool-intro-home-page.png)<!-- do not reuse -->
+
+
+<!-- ------------------------------ -->
+#### Recorded profile timeline
+
+Also use the **Performance** tool to view a recorded performance trace, including: 
+* Record a performance profile.
+* Change capture settings.
+* Analyze a performance report.  Shows information about the performance of the page over the period of time where the recording occurred, such as:
+   * The user interactions that occurred during the recording.
+   * The JavaScript code that ran.
+   * The styling and layout operations that occurred.
+   * Other performance metrics over time, such as network requests and layout shifts.
+
+After you record a performance trace, the performance trace is displayed instead of **Local metrics**.  
+
+
+<!-- ------------------------------ -->
+#### Switching to viewing local metrics or another profile
+
+To switch to displaying the initial, **Local metrics** view, click the **Go back to the live metrics page** (![The "Go back to the live metrics page" (house) icon](./overview-images/house-icon.png)) button.
+
+To switch to displaying a different opened recorded profile timeline,  click the **Show recent timeline sessions** dropdown, to the right of the **Go back to the live metrics page** (![The "Go back to the live metrics page" (house) icon](./overview-images/house-icon.png)) button.
+
+![The Performance tool displaying a recorded profile](./overview-images/performance-tool-intro-profile.png)<!-- do not reuse -->
+
+
+<!-- ====================================================================== -->
+## Open the Performance tool
```

Comment:
It's a little weird to explain how to open the tool _after_ we've spent time talking about the local metrics view, the profile view, and how to switch between the two.

Can we change this to be a h4 inside Overview instead?

- h2 Overview
  - h4 Open the Performance tool
    - h6 Using the Command Menu
  - h4 Local metrics for page interactions
  - h4 Recorded profile timeline
  - h4 Switching to viewing local metrics or another profile

Reply:
![image](https://github.com/user-attachments/assets/0fd09526-57b6-4e84-bee3-08ba06d23214)